### PR TITLE
Add flan-t5-xxl and stable-diffusion-upscaler notebooks

### PR DESCRIPTION
**Pull request by xiangxu-google**
**REQUIRED:** Add flan-t5-xxl and stable-diffusion-upscaler notebooks

**REQUIRED:** Fill out the below checklists or remove if irrelevant

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ y] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [y ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb [hunk 1]
```diff
@@ -0,0 +1,447 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: <p>## Initialize Vertex AI SDK for Python</p><p> </p><p>Initialize the Vertex AI SDK for Python for your project and corresponding bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: <p>Explain what the functions do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    aiplatform.init(project=PROJECT_ID, location=REGION, staging_bucket=GCS_BUCKET)
</pre><p>redundant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: <p>Combine the 2 text boxes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.        display_name=job_name, container_uri=TRAIN_DOCKER_URI, command=["torchrun"]
</pre><p>one parameter per line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: <p>delete the job?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb [hunk 2]
```diff
@@ -0,0 +1,488 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: <p>Combine this and next text cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: <p>## Initialize Vertex AI SDK for Python</p><p> </p><p>Initialize the Vertex AI SDK for Python for your project and corresponding bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: <p>Explain each function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: <p>explain what the code does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: <p>one param per line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: <p>Add heading: ## Cleanup</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

File: notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb (hunk 1 above)
Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: Done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: Added comments for functions below

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: Done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: done

[the remark this answers is not in the document]
Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: done

[the remark this answers is not in the document]
Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb:1]: It deletes the model deployed above.

File: notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb (hunk 2 above)
Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: Next cell is a new section with a header.

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: Added comments for functions below.

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: done

Comment (xiangxu-google) [reply notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_upscaler.ipynb:1]: done

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**